diff --git a/dist/index.js b/dist/index.js
index 185b386841dd6e7366403590da75419e7ccaf180..4711ec994aa2252267e3c8f49e9ba69ef29dac97 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -2562,6 +2562,10 @@ class Ye {
     if (!n && this.current === s)
       return;
     const r = this.current;
+    const isLikePrev = (prev, next) => prev.search === next.search && prev.pathname === next.pathname && prev.hash === next.hash
+    if (isLikePrev(s, r)) {
+      return
+    }
     if (this.index !== t) {
       const i = this._index;
       this._index = t, this.attached && i > 0 != t > 0 && this.sync();
